stan/math/prim/fun/boost_policy.hpp

```diff
@@ -15,13 +15,19 @@ namespace math {
  * (1) overflow errors return error numbers on error.
  * (2) pole errors return error numbers on error.
  * (3) doubles passed to Boost are not promoted to long double.
+ *
+ * The policy is equipped with an optional generic argument B controlling the
+ * precision in some functions. If set to 0, the maximum precision available
+ * in the type being used is demanded from Boost. Otherwise, it correspond to
+ * the approximately B-bit precision, i.e. for trading speed for accuracy.
  */
+template <int B = 0>
```

Comment:
We do not need it. Nevertheless, I would say we may keep it that way just for making changes easier in future: The boost policy is now everywhere registered as generic struct with default arguments with the `boost_policy<>()` rather than `boost_policy()` syntax. Adding other default template arguments is now far easier than before, as long as the C++ version is not lifted to C++17 where both expressions are equal. But personally, I do not care.